### PR TITLE
(maint) use systemd service provider for redhat 8

### PR DESCRIPTION
**Pull request by gimmyxd**

### CODE REVIEW

**Comment by gimmyxd** [comment]
Travis build is failing on ruby 1.9.3 with: mri_22` is not a valid platform. Any idea about that?

**Comment by ekinanp** [comment]
@gimmyxd As for the Travis failures, have you fetched the recent 5.5.x changes?

**Comment by gimmyxd** [comment]
passing adhoc build: https://jenkins-master-prod-1.delivery.puppetlabs.net/view/Adhoc/job/platform_puppet-agent-extra_puppet-agent-integration-suite_adhoc-ad_hoc/438/

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: ruby
 sudo: false
 bundler_args: --without development extra packaging
+before_install:
+  - "gem update bundler"
```
Comment (gimmyxd) [inline .travis.yml]: @joshcooper @Iristyle 
Seems like travis has issues again with the bundler version on ruby 1.9.3. I don't have a better workaround for this yet.
```
[0K$ ruby --version
ruby 1.9.3p551 (2014-11-13 revision 48407) [x86_64-linux]
$ rvm --version
rvm 1.29.3 (latest) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
$ bundle --version
Bundler version 1.7.6
$ gem --version
2.4.3
travis_fold:end:ruby.versions
[0K
travis_fold:start:before_install
[0Ktravis_time:start:13833e49
[0K$ gem update bundler
Updating installed gems
Updating bundler
Fetching: bundler-1.17.2.gem
```

Comment (joshcooper) [reply .travis.yml]: I'd prefer to not have to do this for every travis run. Is this still necessary? Can we make it conditional so it only affects the ruby 1.9.3 combination?

**Comment by gimmyxd** [comment]
rerun the adhoc build(the last one is not available anymore)
https://jenkins-master-prod-1.delivery.puppetlabs.net/view/Adhoc/job/platform_puppet-agent-extra_puppet-agent-integration-suite_adhoc-ad_hoc/

Can this be merged?

**Pull request merged**